yarn-project/pxe/src/pxe.ts

```diff
@@ -1098,10 +1104,6 @@ export class PXE {
     return decodedEvents;
   }
 
-  async resetNoteSyncData() {
```

Comment:
This was unused so I dropped it. It's only used by the internal PXE synchronizer so it shouldn't be exposed.

Reply:
I don't think anyone is using it and invoking the method won't get you out of #15753, because the problem is that your index "got outside" of the window. Resetting them won't unstuck you (nor cleaning PXEs db entirely)

Reply:
In that case will not revert this change and all of Nico's feedback on this PR is now addressed.